docs/cluster-agent/GETTING_STARTED.md

```diff
@@ -0,0 +1,225 @@
+# Datadog Cluster Agent - DCA | Getting Started
+
+## Introduction
+
+This document aims to get you started upon the Datadog Cluster Agent. Refer to the [Datadog Cluster Agent - DCA | User Documentation](README.md) for more context about the Datadog Cluster agent. For more technical background, refer to the [Datadog Cluster Agent in Containerized environments](../../Dockerfiles/cluster-agent/README.md) documentation.
+
+## Step by step
+
+**Step 1** - The Datadog Cluster Agent (DCA) needs a proper RBAC to be up and running, first create its:
+
+* Service Account
+* Cluster Role
+* Cluster Role Binding
+
+Those can be found in the [Datadog Cluster Agent Rbac](../../Dockerfiles/manifests/cluster-agent/rbac/rbac-cluster-agent.yaml).
+
+**Step 2** - Run: `kubectl apply -f Dockerfiles/manifests/cluster-agent/rbac-cluster-agent.yaml` from the datadog-agent directory.
+
+**Step 3** - Depending on whether you are relying on a secret to secure the communication between the Node Agent and the Cluster Agent, either:
+
+* Create a secret
+* Set an environment variable
+
+**Step 3.1** - To create a secret, create a 32 characters long base64 encoded string: `echo -n <32_CHARACTERS_LONG_TOKEN> | base64`
+and use this string in the `dca-secret.yaml` file located in the [manifest/cluster-agent/](../../Dockerfiles/manifests/cluster-agent/dca-secret.yaml) directory. Alternately run this one line command: `kubectl create secret generic datadog-auth-token --from-literal=token=<32_CHARACTERS_LONG_TOKEN>`.
+
+**Step 3.2** - Upon creation, refer to this secret with the environment variable `DD_CLUSTER_AGENT_AUTH_TOKEN`  in the manifest of the cluster agent as well as in the manifest of the Agent!
+
+If you are using the secret:
+
+```yaml
+          - name: DD_CLUSTER_AGENT_AUTH_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: datadog-auth-token
+                key: token
+
+```
+ 
+Or otherwise:
+ 
+```yaml
+          - name: DD_CLUSTER_AGENT_AUTH_TOKEN
+            value: "<32_CHARACTERS_LONG_TOKEN>"
```

Comment:
Warn the user: this is readable in the PodSpec. So a non protected `/pods` leaks the data